AlphaWallet/Transactions/ViewModels/TransactionDetailsViewModel.swift

```diff
@@ -54,9 +54,22 @@ struct TransactionDetailsViewModel {
     var shareAvailable: Bool {
         return detailsAvailable
     }
+    
+    var addressCopiedText: String {
+        return R.string.localizable.requestAddressCopiedTitle()
+    }
 
     var detailsURL: URL? {
-        return ConfigExplorer(server: server).transactionURL(for: transaction.id)
+        return ConfigExplorer(server: server).transactionURL(for: transaction.id)?.url
+    }
+    
+    var detailsButtonText: String {
+        if let name = ConfigExplorer(server: server).transactionURL(for: transaction.id)?.name {
+            let value = R.string.localizable.viewIn()
```

Comment:
- [ ] You can set the localisation string as this instead:

```
"view in" = "View in %";
```

And `R` will generate a function with an argument instead, so you can do:

```
return R.string.localizable.viewIn(name)
```